components/devtools/actors/inspector/page_style.rs

```diff
@@ -125,13 +103,15 @@ impl Actor for PageStyleActor {
 
     /// The page style actor can handle the following messages:
     ///
-    /// - `getApplied`: Returns the applied styles for a node, placeholder
+    /// - `getApplied`: Returns the applied styles for a node, they represent the explicit css
+    /// rules set for them, both in the style attribute and in stylesheets.
     ///
-    /// - `getComputed`: Returns the computed styles for a node, placeholder
+    /// - `getComputed`: Returns the computed styles for a node, these include all of the supported
+    /// css properties calculated values.
     ///
-    /// - `getLayout`: Returns the box layout properties for a node, placeholder
+    /// - `getLayout`: Returns the box layout properties for a node.
     ///
-    /// - `isPositionEditable`: Informs whether you can change a style property in the inspector
+    /// - `isPositionEditable`: Informs whether you can change a style property in the inspector.
     fn handle_message(
```

Comment:
This getting quite large. Would it be possible to split it into methods -- one per message?